modules/osci/Makefile

```diff
@@ -46,146 +46,107 @@ OSCI_GIT_USER_NAME ?= "ACM CICD"
 OSCI_GIT_USER_EMAIL ?= "acm-cicd@redhat.com"
```

Comment:
Could this be a secret, or a part of configuration as opposed to committed to the repo?

Reply:
It could be a secret, but I think that would be overkill. If it were part of the configuration, then we'd be relying on everyone to add it correctly to their configs and to update it everywhere if it changes.

Any reason for not having it here? It has it in one place where it's easy to change if needed.

Reply:
We'll get flagged by JonBot, but it's not the end of the world.

Reply:
It's probably already been flagged since it was in the previous version of osci/publish.